apps/daimo-mobile/src/view/screen/send/SendTransferButton.tsx

```diff
@@ -92,11 +93,22 @@ export function SendTransferButton({
         chainGasConstants: account.chainGasConstants,
       };
 
-      // Swap and transfer if outbound coin is different than home coin.
-      if (isSwap) {
-        console.log(`[ACTION] sending via swap with route ${route}`);
+      // TODO: handle case with swap and bridge
+      // TODO: check against home chain instead of hardcoding base
+      if (toChain.chainId !== base.chainId) {
+        console.log(`[ACTION] sending via FastCCTP to chain ${toChain.name}`);
+        return opSender.sendUsdcToOtherChain(
+          recipient.addr,
+          toChain,
+          dollarsStr,
+          opMetadata,
+          memo
+        );
+      } else if (isSwap) {
```

Comment:
would add back this log for visibility (in case route fails for some reason)

`console.log(`[ACTION] sending via swap with route ${route}`);`